packages/terra-pills/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "terra-pills",
```

Comment:
Lets make this a scoped package
```suggestion
  "name": "@cerner/terra-pills",
```

Reply:
changed here 5158250e39187534d43a615c7ee12bd447c7ab6b

Reply:
changed here 5158250e39187534d43a615c7ee12bd447c7ab6b